src/soter/boringssl/soter.mk

```diff
@@ -15,6 +15,10 @@
 #
 
 SOTER_SRC += $(wildcard $(CRYPTO_ENGINE)/*.c)
+SOTER_AUD_SRC += $(wildcard $(CRYPTO_ENGINE)/*.c)
+SOTER_AUD_SRC += $(wildcard $(CRYPTO_ENGINE)/*.h)
+SOTER_FMT_SRC += $(wildcard $(CRYPTO_ENGINE)/*.c)
+SOTER_FMT_SRC += $(wildcard $(CRYPTO_ENGINE)/*.h)
```

Comment:
why are we doing all these variable copies for the same data?

Reply:
Consistency. Should I just replace this with the following?

```make
CRYPTO_ENGINE_SOURCES = $(wildcard $(CRYPTO_ENGINE)/*.c)
CRYPTO_ENGINE_HEADERS = $(wildcard $(CRYPTO_ENGINE)/*.h)

SOTER_SRC += $(CRYPTO_ENGINE_SOURCES)
SOTER_AUD_SRC += $(CRYPTO_ENGINE_SOURCES) $(CRYPTO_ENGINE_HEADERS)
SOTER_FMT_SRC += $(CRYPTO_ENGINE_SOURCES) $(CRYPTO_ENGINE_HEADERS)
```

Reply:
we already have some variables, why create more aggregate variables? Our format target can just depend on `$(SOTER_SRC) $(CRYPTO_ENGINE_SOURCES) ..`